### PR TITLE
Cocktail details

### PR DESCRIPTION
**Pull request by HeloiseFerla**

### CODE REVIEW

File: src/components/CocktailDetail/style.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  div.topLeft p {
+    margin-top: 1rem;
+    padding: 1rem 0;
+  }
+
+  div.bottomRight {
+    height: 100%;
+  }
+}
```
Comment (PierreGoe) [inline src/components/CocktailDetail/style.css:87]: maybe use Grid css in this case is better but it's good

**Review by PierreGoe — DISMISSED** [review]

File: src/components/CocktailDetail/index.jsx [hunk 2]
```diff
@@ -1,3 +1,55 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import './style.css';
+import { useParams } from 'react-router-dom';
+
 export default function CocktailDetail() {
-  return 'Work in progress';
+  const { cocktailName } = useParams();
+  const [cocktail, setCocktail] = useState([]);
```
Comment (Hacene010) [inline src/components/CocktailDetail/index.jsx]: If your state is an array it should be plural

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Take a look on the comment and fix the error plz

**Review by PierreGoe — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]
ok for me ; )

**Pull request merged**